static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -197,7 +197,7 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
     );
   }
 
-  if (!currentPlatform || !hasOnboardingContents) {
+  if (!currentPlatform || (!hasOnboardingContents && !newOnboarding)) {
```

Comment:
extra check here because `astro` doesn't have onboarding contents in our old docs

Reply:
It's probably because the astro sdk is so new the instructions are missing. They should be `npx astro add @sentry/astro` and all the rest is the same as usual. 

https://vanguard.getsentry.net/p/clpldeg900000s60qdicskfkm has some context on what's new and coming.

Reply:
so i'd take out the new checks

Reply:
got it. i added some new docs in `sentry-docs`: https://github.com/getsentry/sentry-docs/pull/8649